distribution/docs/src/main/resources/content/_configuring/configuring-guest-access.adoc

```diff
@@ -8,10 +8,11 @@
 == {title}
 
 Unauthenticated access to a secured ${branding} system is provided by the *Guest* user.
-By default, the Search UI allows guest access as part of the karaf security realm.
+By default, ${branding} allows guest access as part of the karaf security realm.
 
-These instructions assume that all security components are running on the same physical or virtual machine.
-For installations where some or all of these components reside on different network locations, adjust accordingly.
+Because ${branding} does not know the identify of a Guest user, it cannot assign security attributes to the Guest.
+The administrator must configure the attributes and values (i.e. the "claims") to be assigned to Guests.
+The Guest Claims become the default attributes for every user, both authenticated and unauthenticated.
```

Comment:
❓ Is this sufficiently clear that the user will *always* get these attributes? In other words, you cannot have a user claim that is more restrictive than the guest claim, the guest claim will grant them access anyways.

Reply:
added.